api/HTMLElement.json

```diff
@@ -1079,54 +1079,6 @@
           }
         }
       },
-      "forceSpellCheck": {
```

Comment:
No trace of this in Chromium, Gecko or WebKit. In WebKit it appears in some performance tests, seemingly copied from the HTML spec where this once appeared:
https://github.com/whatwg/html/commit/4a0651c9d016dbf3a9431679248509baae6c20b3
https://github.com/whatwg/html/commit/c79ad811fb8b650c6079960642b86fe01effc3f7